packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -189,8 +189,11 @@ function useProcessMedia({
     ({ resource: oldResource, start, end }) => {
       const { src: url, mimeType, poster } = oldResource;
 
+      // If we already have an original ID, use that instead.
       const trimData = {
-        original: oldResource.id,
+        original: oldResource.trimData?.original
+          ? oldResource.trimData.original
+          : oldResource.id,
```

Comment:
No, the very original can't have trim data but the `resource` is modified as needed when calling `trimExistingVideo` so that if an `original` exists for the video that's being trimmed, the very original ID (`oldResource.trimData.original`) would be available. 

To make things more clear, perhaps we should change `trimExistingVideo` instead to accept the needed value as a separate param instead.